com.unity.shadergraph/Documentation~/Texel-Size-Node.md

```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-Returns the **Width** and **Height** of the texel size of **Texture 2D** input. Uses the built in variable `{texturename}_TexelSize` to access special properties of a **Texture 2D**.
+Returns the **Width** and **Height** of the texture size of **Texture 2D** input as well as the **Width** and **Height** of the texel size. Uses the built in variable `{texturename}_TexelSize` to access special properties of a **Texture 2D**.
```

Comment:
Returns the texel resolution of the Texture 2D input, as well as the size of each texel in the texture.

Reply:
Do you think texel needs a definition?   Not sure how common that term is.